### PR TITLE
add language code command line option to management command.

### PR DESCRIPTION
**Pull request by bdon**
to set the language code cookie from the command line e.g. 

    python manage.py statictemplate --language-code zh-hans index.html

### CODE REVIEW

File: statictemplate/management/commands/statictemplate.py [hunk 1]
```diff
@@ -82,10 +82,15 @@ class Command(BaseCommand):
...
                     help='Output file'),
+        make_option('--language-code', '-l',
+                    action='store',
+                    dest='language_code',
+                    help='Language Code')
     )
 
     def handle(self, template, language=None, extra_request=None, **options):
         request = {}
+        language = options.get('language_code',False)
```
Comment (yakky) [inline statictemplate/management/commands/statictemplate.py]: Fix style here (``get('language_code', False)`` not ``get('language_code',False)``

**Comment by bdon** [comment]
amended

**Comment by yakky** [comment]
@bdon thanks. Still figuring out where the coverage decrease come from ...

**Comment by yakky** [comment]
@bdon could you investigate the test failures? thanks

**Comment by bdon** [comment]
yeah, forgot and pushed another commit onto this branch - i reverted back to the original

**Comment by yakky** [comment]
@bdon thanks, merging this!

**Pull request merged**